### ISSUE TITLE
Chrome instances pile up when running several tests on RIDE

### ISSUE THREAD

**Issue by rdguerra91** [robotframework/RIDE#2834]
I have about 60 test suite running tests on a static website on Chrome, running in RIDE.
In order for the test to be accomplished I use Robot / Selenium to open / close web browser in between tests and do several actions.

My issue is whenever I close all web browsers on the Test Teardown, sometimes there are "rogue" chrome processes used during the tests that don't get killed/stopped as they should. 
So the more test I run, the more Chrome processes pile up, until my machine comes to an almost stand still due to lack of resources.

Even closing RIDE process on task manager doesn't solve the issue, so I have to terminate all chrome processes in order to regain full performance, even in between tests.

Running the same tests in a pipeline, or even running them via command line, bring no issues regarding the rogue Chrome processes, so I'm just kind of confused in what's maybe causing this.

PS: We recently upgraded from Robot Framework 6.x and this didn't happen.

We run: 

- Robot Framework (v7.0)
- RF-Selenium Library (v6.2.0)
- Selenium (v4.18.1)
- RIDE (v2.1dev58)

**Comment by JFoederer** [comment]
Hi @rdguerra91, since this issue is entered for RIDE can you please run the same suite directly with a robot command (without RIDE) and confirm the issue does not show up there? That would better isolate the issue and confirm that it is only triggered when executing the tests from RIDE.

**Comment by HelioGuilherme66** [comment]
@rdguerra91 You can create a Macro (Run Anything) to kill all Chromedriver instances, or have a final Test Suite Teradown that does this.
It is strange that RIDE is the cause of rogue Chromedriver processes. Most probably is that the Teardowns are not being executed, and the processes not being closed normally.

RIDE uses the normal call of Robot.

**Comment by rdguerra91** [comment]
I have been searching a bit more and it can be related to Selenium handling of driver.quit() and driver.close() leaving behind one process hanging.

I tried via cmdline and the issue persists, I steer away from RIDE in my investigations.

All my test teardown include the "Close All Browsers" KW, that should in theory kill all chrome related processes.
I tracked down the issue to Ride, or I thought I did, due to the pipeline being ran daily without any apparent issue, but I'm looking at this from a bad angle probably, sorry for bothering!
